services/rss_rendering_service.js

```diff
@@ -0,0 +1,68 @@
+var moment = require('moment');
+moment().format();
+module.exports = function RSSRenderingServiceModule(pb) {
+  const WORDS_PER_PREVIEW = 24; 
+  
+  function RSSRenderingService(options){
+    if (options) {
+      this.site = options.site || pb.SiteService.GLOBAL_SITE;
+    } else {
+      this.site = pb.SiteService.GLOBAL_SITE;
+    }
+  }
+
+  RSSRenderingService.init = function(cb){
+    pb.log.debug("RSSRenderingService: Initialized");
+    cb(null, true);
+  };
+
+  RSSRenderingService.getName = function(){
+    return "rssRenderingService";
+  };
+
+  RSSRenderingService.prototype.render = function(cb) {
+    var self = this;
+    if(pb.PluginService.isActivePlugin('pencilblue_rssfeeder', self.site)) {
```

Comment:
You do not need to check if you are active.  Rss feeder is the active plugin therefore does not need to be checked no?  Probably just a carry over from when this lived in CBC.